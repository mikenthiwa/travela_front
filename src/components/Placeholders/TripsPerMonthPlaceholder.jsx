import React from 'react';
import ContentLoader from 'react-content-loader';

const TripsPerMonthPlaceholder = props => (
  <ContentLoader
    height={250}
    width={500}
    speed={1.5}
    primaryColor="#dfdfdf"
    secondaryColor="#ffffff"
    {...props}
  >
    <rect x="7.33" y="3.61" rx="0" ry="0" width="172.54" height="11.52" />
    <rect x="394.63" y="1.61" rx="0" ry="0" width="132.99" height="28.12" />
    <rect x="7.67" y="75.61" rx="0" ry="0" width="525.5" height="0.85" />
    <rect x="7.63" y="51.61" rx="0" ry="0" width="71.1" height="10.66" />
    <rect x="228.63" y="50.74" rx="0" ry="0" width="112.91" height="9.79" />
    <rect x="3.63" y="89.61" rx="0" ry="0" width="530" height="171" />

  </ContentLoader>
);

export default TripsPerMonthPlaceholder;
