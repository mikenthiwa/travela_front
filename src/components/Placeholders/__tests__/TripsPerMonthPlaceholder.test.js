import React from 'react';
import TripsPerMonthPlaceholder from '../TripsPerMonthPlaceholder';


describe('<TripsPerMonthPlaceholder />', () => {
  const wrapper = shallow(<TripsPerMonthPlaceholder />);

  it('should render without crashing', () => {
    const wrapper = shallow(<TripsPerMonthPlaceholder />);
    expect(wrapper).toMatchSnapshot();
  });
});
