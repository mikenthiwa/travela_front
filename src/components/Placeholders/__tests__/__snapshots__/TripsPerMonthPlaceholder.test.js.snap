// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TripsPerMonthPlaceholder /> should render without crashing 1`] = `
<ContentLoader
  height={250}
  primaryColor="#dfdfdf"
  secondaryColor="#ffffff"
  speed={1.5}
  width={500}
>
  <rect
    height="11.52"
    rx="0"
    ry="0"
    width="172.54"
    x="7.33"
    y="3.61"
  />
  <rect
    height="28.12"
    rx="0"
    ry="0"
    width="132.99"
    x="394.63"
    y="1.61"
  />
  <rect
    height="0.85"
    rx="0"
    ry="0"
    width="525.5"
    x="7.67"
    y="75.61"
  />
  <rect
    height="10.66"
    rx="0"
    ry="0"
    width="71.1"
    x="7.63"
    y="51.61"
  />
  <rect
    height="9.79"
    rx="0"
    ry="0"
    width="112.91"
    x="228.63"
    y="50.74"
  />
  <rect
    height="171"
    rx="0"
    ry="0"
    width="530"
    x="3.63"
    y="89.61"
  />
</ContentLoader>
`;
